src/timer.js

```diff
@@ -0,0 +1,14 @@
+let timer = function(i) {
```

Comment:
why a let? and why not a fat arrow function?

Reply:
`i` is not a great variable name. also if you default this to `30` you could improve the code below